docs/porting/overview.md

```diff
@@ -1,5 +1,72 @@
+
+Your code could be a good candidate for porting to .NET Core if...
+
+- Your code is a library and you want to expand its reach
+- You have a compelling business case to run code on more than just Windows
+- You want to take advantage of app-local deployment and build self-contained console applications
+- You have expertise on non-Windows platforms you wish to leverage
+- You're looking for an opportunity to implement an architectural change for your services (e.g. a monolithic service to microservices deployed to containers)
+- You love new technology and want to cut your teeth on .NET Core
+
+Your code may not be a good candidate for porting if...
+
+- Your code is heavily tied to a Windows GUI or depends on Windows-specific features
+- You have no business needs or expertise which would warrant running on more than Windows
+- You don't have a need for app-local deployment or self-contained console apps
+- You're happy with your code running on .NET Framework as it is today
+
+If it makes sense to port at this time, then read on!
+
+## Overview of the Porting Process
+
+The recommended process for porting follows a series of steps.  Here's a short overview of the process in general.  There will be other specific steps to take when porting libraries, apps and services.
+
+1. Identify and account for your third-party dependencies.
+
+   This will involve understanding what your 3rd party dependencies are, how you depend on them, how to see if they also run on .NET Core, and steps you can take if they don't.
+   
+2. Change all projects you wish to port to target .NET Framework 4.6.1.
+
+   This ensures that you can use API alternatives for .NET Framework-specific targets in the cases where .NET Core can't support a particular API.
+   
+3. Use the API Portability Analyzer to analyze your assemblies and develop a plan based on its results.
+
+   The API Portability Analyzer tool will analyze your compiled assemblies and generate a report which shows a high-level portability summary and a breakdown of each API you're using that isn't available on .NET Core.  You can use this report alongside an analysis of your codebase to develop a plan for how you'll port your code over.
+   
+4. Port your tests over to xUnit and NUnit.
+
+   MSTest doesn't support .NET Core, so you'll have to use either xUnit or NUnit to test your .NET Core targets.  Because porting to .NET Core is such a big change to your codebase, it's highly recommended to get your tests ported so that you can run tests as you port projects over.
+   
+5. Determine which project system in Visual Studio is appropriate for your needs.
+
+   There are now two project systems in Visual Studio which support .NET Core: the traditional `csproj` project system and the new `xproj` project system.  Both offer capabilities that the other does not, so it's critical that you pick the correct project system.  This will be covered in other articles.
```

Comment:
I decided to go with talking about these in terms of the project system, not particular projects you can create with a specific project system.  I also chose to call it the "traditional project system" because csproj is only one of the csproj/vbproj/fsproj family that share the same functionality.